driver/docker-container/driver.go

```diff
@@ -217,7 +217,7 @@ func (d *Driver) wait(ctx context.Context, l progress.SubLogger) error {
 			select {
 			case <-ctx.Done():
 				return context.Cause(ctx)
-			case <-time.After(time.Duration(try*120) * time.Millisecond):
+			case <-time.After(d.Timeout / time.Microsecond):
```

Comment:
see https://github.com/docker/buildx/pull/2586/files#diff-403916c02204229a19313f586f2fffa2536dde693a6e625bc4525f25be068ec9

Reply:
So it looks like before it did a linear backoff with step=120ms:

try 1: After 120ms
try2: After 240ms
…
try 15: After 1,800ms

total: 14,400ms or 14.4s


if the new Timeout is the total time and we want to keep the linear backoff then I think the math should be:

total = 15*(1*step+15*step)/2
(total*2)/(15*16) = step

I implemented somethng like that.